intercom-java/src/main/java/io/intercom/api/Conversation.java

```diff
@@ -272,6 +275,14 @@ public ConversationPartCollection getConversationPartCollection() {
         return conversationPartCollection;
     }
 
+    public TagCollection getTagCollection() {
+        if (tagCollection == null) {
```

Comment:
I thought initially this was an issue with potential infinite loop but actually the opposite. Mentioned it to @mmartinic and he noted that this is used for list as well as single convos so there could be a case where there are no tags returned for listing class.